src/Symfony/Component/Routing/Matcher/Dumper/PhpMatcherDumper.php

```diff
@@ -358,8 +370,19 @@ private function compileDynamicRoutes(RouteCollection $collection, bool $support
             }
             if ($matchHost) {
                 $code .= "\n                .')'";
+                $state->regex .= ')';
+            }
+            $rx = ")$}{$modifiers}";
+            $code .= "\n                .'{$rx}',";
+            $state->regex .= $rx;
+
+            // if the regex is too large, throw a signaling exception to recompute with smaller chunk size
```

Comment:
curious... why not have (reasonably large) fixed limit instead? is it variable in case of PCRE? It could avoid any recomputing needed no?

Reply:
The large fixture is a result of this choice i guess.. in theory that could break in different envs.. :thinking: should it be mocked?

Reply:
Not sure we should care about such concerns.